### PR TITLE
Feature/personal profile language

### PR DESCRIPTION
**Pull request by WalterGoldenOwl**
Description Pull Request:
## Todo
- Apply UI and logic CRUD language

## Task
Link of task:  https://trello.com/c/dNNgtgqd

## Screenshot
Screenshot_1697703398
Screenshot_1697703401
Screenshot_1697703429

## Video

https://github.com/dustin-goldenowl/jobspot/assets/144195318/54658acf-9c03-4706-a235-26ff1c7e6dbb

### CODE REVIEW

File: lib/src/presentations/add_language/screen/add_language_view.dart [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+          decoration: BoxDecoration(
+            color: Colors.white,
+            borderRadius: BorderRadius.circular(20),
+          ),
+          padding: const EdgeInsets.symmetric(horizontal: 15, vertical: 20),
+          child: Column(
+            children: [
+              _buildLanguage(),
+              const SizedBox(height: 20),
+              const Divider(color: Color(0xFFDEE1E7), thickness: 0.5),
```
Comment (dustin-goldenowl) [inline lib/src/presentations/add_language/screen/add_language_view.dart]: Put in color resource

File: lib/src/presentations/add_language/screen/add_language_view.dart [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+            color: Colors.white,
+            borderRadius: BorderRadius.circular(20),
+          ),
+          padding: const EdgeInsets.symmetric(horizontal: 15, vertical: 20),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              _buildOral(context),
+              const SizedBox(height: 20),
+              const Divider(color: Color(0xFFDEE1E7), thickness: 0.5),
```
Comment (dustin-goldenowl) [inline lib/src/presentations/add_language/screen/add_language_view.dart]: The same above

File: lib/src/presentations/add_language/widgets/choose_level_language_dialog.dart [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+          const SizedBox(height: 20),
+          ...List.generate(
+            11,
+            (index) => GestureDetector(
+              onTap: () => setState(() => level = index),
+              behavior: HitTestBehavior.opaque,
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Text("Level $index"),
```
Comment (dustin-goldenowl) [inline lib/src/presentations/add_language/widgets/choose_level_language_dialog.dart]: put in app_en.arb

File: lib/src/presentations/add_language/screen/add_language_view.dart (hunk 1 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/add_language/screen/add_language_view.dart]: I fixed it

File: lib/src/presentations/add_language/widgets/choose_level_language_dialog.dart (hunk 3 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/add_language/widgets/choose_level_language_dialog.dart]: I fixed it

File: lib/src/presentations/add_language/screen/add_language_view.dart (hunk 2 above)
Comment (WalterGoldenOwl) [reply lib/src/presentations/add_language/screen/add_language_view.dart]: I fixed it

**Pull request merged**